content/sdk/10.js/00.ethers/00.index.md

```diff
@@ -17,16 +17,41 @@ Take a look at the [migration guide](/sdk/js/ethers/v6/migration) if you are mig
 
 ### Installation
 
-To install SDK for zkSync Era, run the command below in your terminal.
+To install SDK for zkSync Era, follow the process below:
+
+#### Step 1: Create a Folder
+
+1. Open your terminal or command prompt.
+2. Navigate to the directory where you want to create your project folder.
+3. Create a new folder for your project:
+
+    ```sh
+    mkdir zkSyncProject
+    cd zkSyncProject
+    ```
+
+#### Step 2: Initialize a Node.js Project
+
+1. Inside your project folder, initialize a new Node.js project:
+
+    ```sh
+    npm init -y
+    ```
+
+    This will create a `package.json` file.
+
+#### Step 3: Install Required Packages
```

Comment:
These number of steps seems unnecessary? They could have their project set up in a variety of ways and many might be looking to just integrate it into an existing project. We should just state that they install npm packages in a nodejs project at the most. The most important steps a user wants to see are just the install commands.

Reply:
Number steps are important for organizing documentation. 
Also, its helpful for newbies who wants to start using zksync-era except we are not targeting newbies.
I have re-arranged the steps.

Reply:
I also agree that numbering is very useful for readability. What I meant was removing the steps on how to setup a project. As you said, we are not targeting newbies. They should know how to setup and manage their own projects, the primary purpose of this page is instructing them what packages to install. 
Also, the instructions you are providing for setting up a project are also primarily for linux systems, it may not work on windows systems.

If we want to demonstrate how to create a project and install packages, we can do so in our Community Code project.

Reply:
I have already removed the steps and updated it